test/unit/typings-compatibility-test.ts

```diff
@@ -23,42 +23,14 @@ type TestDef = {version: string; minTarget: string; requiredLibs?: string[]}
 
 // NOTE tsc version 2.9.x (and lower) will not work with current typings!
 const tsVersions: TestDef[] = [
-  // typescript 3.0.x - 3.4.x:
+  // typescript 3.0.x - 3.3.x:
   //  must either have a target that supports AsyncIterators,
   //  or include a typings library that supports AsyncIterator
-  {
-    version: "3.0.x",
-    minTarget: "es3",
-    requiredLibs: ["es2015", "ESNext.AsyncIterable"],
-  },
-  {
-    version: "3.1.x",
-    minTarget: "es3",
-    requiredLibs: ["es2015", "ESNext.AsyncIterable"],
-  },
-  {
-    version: "3.2.x",
-    minTarget: "es3",
-    requiredLibs: ["es2015", "ESNext.AsyncIterable"],
-  },
-  {
-    version: "3.3.x",
-    minTarget: "es3",
-    requiredLibs: ["es2015", "ESNext.AsyncIterable"],
-  },
-  {
-    version: "3.4.x",
-    minTarget: "es3",
-    requiredLibs: ["es2015", "ESNext.AsyncIterable"],
-  },
-  {version: "3.0.x", minTarget: "esnext"},
-  {version: "3.1.x", minTarget: "esnext"},
-  {version: "3.2.x", minTarget: "esnext"},
-  {version: "3.3.x", minTarget: "esnext"},
-  {version: "3.4.x", minTarget: "esnext"},
-
-  // typescript 3.5.x - 3.7.x:
+  // TODO might support older TypeScript
```

Comment:
Can we track this in an issue instead?

Reply:
To be honest, I don't think this is necessary at all for two reasons. First, rarely people build TypeScript from the source when the package is distributed, and they use the generated JavaScript. Second, if anyone would want to do that anyways, it is much easier to upgrade older TypeScript to newer ones rather than trying to support older ones.